APMrover2/test.cpp

```diff
@@ -214,8 +214,8 @@ int8_t Rover::test_wp(uint8_t argc, const Menu::arg *argv)
 {
     delay(1000);
 
-    cliSerial->printf("%u waypoints\n", (unsigned)mission.num_commands());
-    cliSerial->printf("Hit radius: %f\n", (double)g.waypoint_radius.get());
+    cliSerial->printf("%u waypoints\n", static_cast<unsigned int>(mission.num_commands()));
```

Comment:
@lucasdemarchi I change unsigned to unsigned int but I read on I2C fix that unsigned isn't the same than unsigned int. Should I revert that?